documentation/src/main/asciidoc/topics/con_near_caches.adoc

```diff
@@ -0,0 +1,35 @@
+[id='near-caches-{context}']
+= Near Caches
+Near caches are local to Hot Rod clients and store recently used data so that every read operation does not need to traverse the network, which significantly increases performance.
+
+Near caches are populated on calls to `get()` or `getVersioned()`.
+When entries are updated or removed in remote caches on {brandname} Servers, those entries are invalidated in the near cache.
+If entries are requested after they are invalidated, clients must retrieve them from the remote caches again.
+
+Near caches are also cleared when clients fail over to different servers.
+
+.Bounded near caches
+
+You should always use bounded near caches by specifying the maximum number of entries they can contain.
+When near caches reach the maximum number of entries, eviction automatically takes place to remove older entries.
+This means you do not need to manually keep the cache size within the boundaries of the client JVM.
+
+[IMPORTANT]
+====
+Do not use maximum idle expiration with near caches because near-cache reads do not propagate the last access time for entries.
+====
+
+.Bloom filters
+
+Bloom filters optimize performance for write operations by reducing the total number of invalidation messages.
+
+Bloom filters use a client listener that is registered on the {brandname} Server to keep the near cache in sync.
```

Comment:
The bloom filter has nothing to do with the listener, the near cache itself does that. The bloom filter is an extra thing kept on the server side in addition to the listener that keeps track of what keys have been requested (in a bloom filter). This way if a key is updated that doesn't match the bloom filter the update for the listener isn't sent back to the client.

Reply:
Cheers for clearing that up for me. I've revised this.